src/script/script_instance.hpp

```diff
@@ -25,6 +27,9 @@ class ScriptInstance {
 	friend class ScriptObject;
 	friend class ScriptController;
 
+	typedef std::variant<byte, SQInteger, std::string, SQBool> ScriptDataVariant;
```

Comment:
Isn't the "byte" always a "SQSaveLoadType"?

Reply:
It is, I guess I should move the enum to the header.